src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/ValidationAttributeStore.cs

```diff
@@ -212,12 +213,9 @@ internal bool TryGetPropertyStoreItem(string propertyName, out PropertyStoreItem
 
                 if (_propertyStoreItems == null)
                 {
-                    lock (_syncRoot)
+                    lock (_lock)
```

Comment:
As far as I can tell, this is a buggy change.  Why did you change this?  It's now using a process-wide lock to protect instance state, and it removed the double-check that would ensure that multiple threads always see the same value even in the face of race conditions.